contributors/devel/pull-request-commands.md

```diff
@@ -0,0 +1,89 @@
+# Overview
+
+There are several steps to authoring or reviewing a pull request in Kubernetes.
+Due to limitations on how GitHub permissions can be expressed, the Kubernetes
+authors developed a workflow using comments to run tests and set labels
+used for merging PRs.  Merging a PR requires the following steps to be
+completed before the PR will automatically be merged:
+
+- Signing a CLA
+- Setting release notes
+- Having all e2e tests pass
+- Getting an LGTM from a reviewer
+
+# Master Branch Workflow
+
+## Sign the CLA
+
+If you have not already, the `k8s-ci-robot` will leave a comment with
+instructions on how to sign the cla.
+
+**Important** the email you sign the cla with must be the same email
+attached to the commits in your PR.  If it is not, you may need to change
+the email and repush the commits.
+
+## Set Release Notes
+
+Every PR must tag whether it requires an update to the release notes.  This
+is done by commenting either `/release-note` or `/release-note-none` to set the
+`release-note` or `release-note-none` labels respectively.
+
+Release notes should be present for any PRs with user visible changes such as
+bug-fixes, feature additions, and output format changes.
+
+Specify the release note by adding the following section to the PR description
+with the release note inside of it.  Otherwise the release note will
+be take from the PR name.
+
+\`\`\`release-note
```

Comment:
I think if you indent this it will be a bit clearer:

    ```release-note
    release note here
    ```
And the way to mark empty release notes:

    ```release-note
    NONE
    ```